janusgraph-core/src/main/java/org/janusgraph/graphdb/idmanagement/IDManager.java

```diff
@@ -567,6 +567,33 @@ public long getPartitionedVertexId(long partitionedVertexId, long otherPartition
         return ids;
     }
 
+    /**
+     * Converts a user provided long id into a JanusGraph vertex id. The id must be positive and less than {@link #getVertexCountBound()}.
+     * This method is useful when providing ids during vertex creation via {@link org.apache.tinkerpop.gremlin.structure.Graph#addVertex(Object...)}.
+     *
+     * @param id long id
+     * @return a corresponding JanusGraph vertex id
+     * @see #fromVertexId(long)
+     */
+    public long toVertexId(long id) {
```

Comment:
@amcp There's a test added as part of the commit (see [VertexIDAssignerTest#testCustomIdAssignment](https://github.com/amcp/janusgraph/blob/fixBlueprintsCustomIds/janusgraph-test/src/test/java/org/janusgraph/graphdb/idmanagement/VertexIDAssignerTest.java#L186-L208)).